src/styles/blocks/card.scss

```diff
@@ -0,0 +1,109 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: $white-color;
+
+  transition-duration: $transform-duration;
+
+  &:hover {
+    transform: scale(1.2);
+    .card__product-name {
+      color: #34568b;
+    }
+  }
+
+  &__product-image {
+    display: flex;
+    align-self: center;
+
+    margin-bottom: 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__product-name {
+    width: 166px;
+    margin: 0 0 4px;
+
+    font-weight: 500;
+    font-size: 12px;
+  }
+
+  &__product-code {
+    margin: 0 0 16px;
+
+    color: $secondary-color;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 16px;
+  }
+
+  &__price-label {
+    margin: 0;
+    color: $secondary-color;
+    font-size: 12px;
+  }
+
+  &__price-value {
+    margin: 0;
+    text-align: right;
+    font-size: 16px;
+    font-weight: 700;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 24px;
+  }
+
+  &__reviews-quantity {
+    margin: 0;
+    text-align: right;
+    font-size: 10px;
+    font-style: normal;
+    line-height: 14px;
+  }
+
+  &__buy {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    width: 166px;
+    height: 40px;
+
+    font-weight: 700;
+    line-height: 16px;
+    font-size: 14px;
+    text-align: center;
+    text-transform: uppercase;
+    text-decoration: none;
+    color: $white-color;
+
+    border-radius: 5px;
+    background: $blue-color;
+
+    transition-duration: $transform-duration;
```

Comment:
```suggestion
    transition-duration: $transform-duration;
    &:hover {}
```

Reply:
https://github.com/mate-academy/layout_catalog/pull/3438/files#r1270652138
didn't fix it